### PR TITLE
Enable invalidcommitmsg plugin on k-sigs/contributor-playground

### PR DESCRIPTION
**Pull request by nikhita**
/hold
until the next prow bump so https://github.com/kubernetes/test-infra/pull/12199 is picked up

This PR also adds the `do-not-merge/invalid-commit-message` label.

/assign @cblecker

### CODE REVIEW

File: prow/plugins.yaml [hunk 1]
```diff
@@ -658,6 +658,7 @@ plugins:
   - trigger
 
   kubernetes-sigs/contributor-playground:
+  - invalidcommitmsg
```
Comment (nikhita) [inline prow/plugins.yaml]: I thought it was best to try out the plugin on this repo first. Thoughts?

Also...does this need lazy consensus on the contribex ML?

Comment (mrbobbytables) [reply prow/plugins.yaml]: I don't think it needs lazy consensus, and I think this is a great place to use it :) This is a safe repo for folks to learn good habits and it helps with that. 👍

Comment (cblecker) [reply prow/plugins.yaml]: No consensus needed on the contributor playground.

**Comment by nikhita** [comment]
Prow has been bumped in https://github.com/kubernetes/test-infra/pull/12219. Cancelling the hold.

/hold cancel

**Pull request merged**